int32 addMe
---
bool success
int32 filterID
